src/main/java/bio/terra/cli/command/app/passthrough/Nextflow.java

```diff
@@ -16,11 +19,17 @@
 
   @CommandLine.Mixin WorkspaceOverride workspaceOption;
   @CommandLine.Unmatched private List<String> command = new ArrayList<>();
+  private static final Logger logger = LoggerFactory.getLogger(Nextflow.class);
 
-  /** Pass the command through to the CLI Docker image. */
+  /**
+   * Enable the user's pet SA in the workspace context, then pass the command through to the CLI
+   * Docker image.
+   */
   @Override
   protected void execute() {
     workspaceOption.overrideIfSpecified();
+    String petEmail = Workspace.enablePet();
+    logger.info("Enabled usage of for pet {} for Nextflow", petEmail);
```

Comment:
Often this WSM call will be a no-op, so I think it would be better to either:
- do this here once and then cache that it's already enabled so we don't tack on an extra call to every `nextflow` call
- do this when we fetch the pet SA, which will be out of band of the `nextflow` calls.

The second option means that all pet SAs would be enabled though, not sure if there's any problem with that. I think it would also mean you wouldn't need the separate command to enable a pet.

Reply:
Makes sense. I don't think there's an additional concern with enabling this for every pet, so moving this call to when. we fetch the pet SA credentials seems cleaner.